### DISCUSSION TITLE
Pitlane Analytics

### DISCUSSION THREAD

**Discussion by mohamedhadrami** [br-g/openf1#72] [Show and tell]
Welcome to Pitlane Analytics! 🏎️

Your ultimate F1 companion, offering:

- 🗓️ Season Schedule: Never miss a race.
- 📊 Current Standings: Real-time driver and constructor rankings.
- 📚 Historical Data: Explore F1's rich history.
- 📺 Live Dashboard: Track races live with interactive updates.
- 📉 Telemetry Page: Analyze any race, any driver, any lap.

Check out the screenshots below to see it in action! 📸

### Championship Standings
![pitlaneanalytics vercel app_championship](https://github.com/br-g/openf1/assets/17546031/f708b5eb-f856-43f0-861e-c54ffa2eeb67)

### Live Dashboard
![pitlaneanalytics vercel app_dashboard](https://github.com/br-g/openf1/assets/17546031/06c7875d-bc51-4c6f-95ca-c590e4558281)

### Telemetry Analysis
![pitlaneanalytics vercel app_telemetry](https://github.com/br-g/openf1/assets/17546031/a1fee5ca-1266-4ec5-a71e-13680082469a)

### Historical Archives
![pitlaneanalytics vercel app_archive](https://github.com/br-g/openf1/assets/17546031/638ad30d-bbe0-4ba5-a2de-60ad9dc21e13)

I’d love your feedback to make Pitlane Analytics even better. Dive in at [Pitlane Analytics](https://pitlaneanalytics.vercel.app/) and let me know what you think!

Happy racing! 🏁

**Comment by br-g** [comment]
Very comprehensive website!
Thanks for sharing :wink:

**Comment by adityakotha03** [comment]
This looks amazing, thannks for sharing

**Comment by mohamedhadrami** [comment]
I realized I never made my repo public, so here it is:

https://github.com/mohamedhadrami/pitlane-analytics

Feel free to contribute or use the codebase to further your own projects
